storm-client/src/jvm/org/apache/storm/blobstore/BlobStoreAclHandler.java

```diff
@@ -45,6 +47,7 @@
 public class BlobStoreAclHandler {
     public static final Logger LOG = LoggerFactory.getLogger(BlobStoreAclHandler.class);
     private final IPrincipalToLocal _ptol;
+    private final IGroupMappingServiceProvider _groupMappingProvider;
```

Comment:
We should move away from using '_' as a prefix to member variables.  Please at least fix the new ones to not use it, and if you want to fix all of them that would be great too.